capabilities/analytics/track-event/maps/mixpanel.suma

```diff
@@ -0,0 +1,39 @@
+// API Reference: https://developer.mixpanel.com/reference/events
+
+profile = "analytics/track-event@1.0"
+provider = "mixpanel"
+
+map TrackEvent {
```

Comment:
It is, but because Mixpanel is adding token to different places in body and they say project token isn't secret, I decided to use integration parameter and add it where it is needed.